plugins/org.jkiss.dbeaver.ext.hive/src/org/jkiss/dbeaver/ext/hive/model/HiveSQLDialect.java

```diff
@@ -130,4 +130,9 @@ public char getStringEscapeCharacter() {
     public boolean supportsAlterTableStatement() {
         return false;
     }
+
+    @Override
+    public boolean supportsGroupBy() {
+        return false;
+    }
```

Comment:
It's not true. Hive does support `GROUP BY` statements.

https://cwiki.apache.org/confluence/display/Hive/LanguageManual+GroupBy#LanguageManualGroupBy-Selectstatementandgroupbyclause